package.json

```diff
@@ -32,14 +32,13 @@
   "homepage": "https://github.com/adobe/helix-status#readme",
   "dependencies": {
     "@adobe/helix-log": "^4.0.0",
-    "@adobe/helix-ops": "1.0.0",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.7",
-    "xml-escape": "^1.1.0",
-    "yargs": "^15.0.0"
+    "xml-escape": "^1.1.0"
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.0",
+    "@adobe/helix-ops": "1.0.0",
```

Comment:
Will this still work as a transient dependency for projects depending on helix-status? I put it as production dependency here because I wanted to avoid adding helix-ops to all downstream projects which already include helix-status.

Reply:
no. and I would not rely on this. it is better to add `helix-ops` to all projects that need it as dev dependency. then the relation is clear.

Reply:
i.e. this is exactly what we want to avoid. that changes to helix-ops or helix-status affect directly each others dependents. 

the `helix-ops` dependency is _only_ here, because you need it in the circleci script and **not** because it is related to helix-status.

Reply:
Gotcha